schemas/2018-04-01/Microsoft.Compute.json

```diff
@@ -0,0 +1,432 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2018-04-01/Microsoft.Compute.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Compute",
+  "description": "Microsoft Compute Resource Types",
+  "resourceDefinitions": {
+    "disks": {
```

Comment:
>disks [](start = 5, length = 5)

also add some tests for 2018-04-01 schemas?

Reply:
Yes, still to come